storage/aptosdb/src/lib.rs

```diff
@@ -134,7 +133,7 @@ static ROCKSDB_PROPERTY_MAP: Lazy<HashMap<&str, String>> = Lazy::new(|| {
         "rocksdb.block-cache-pinned-usage",
     ]
     .iter()
-    .map(|x| (*x, format!("aptos_{}", x.replace(".", "_"))))
+    .map(|x| (*x, format!("aptos_{}", x.replace('.', "_"))))
```

Comment:
Why not both to single quotes?  Or is that not supported?

Reply:
the second one expects a string literal or something... rust is amazing